daal4py/sklearn/_utils.py

```diff
@@ -71,11 +71,11 @@ def make2d(X):
 
 def get_patch_message(s):
     if s == "daal":
-        message = "uses Intel® DAAL solver"
+        message = "uses optimized solver"
```

Comment:
My prefer to specify daal4py/oneDAL/Intel® oneDAL here: "uses daal4py solver" / "uses oneDAL solver" . Because it's unclear what "optimized solver" is.

Also, if we change the message we need to change daal4py documentation with description of verbosity: https://intelpython.github.io/daal4py/sklearn.html#scikit-learn-verbose

Reply:
I'm oaky with new "Intel(R) oneAPI Data Analytics Library" variant, thanks. But need to update docs anyway :)